steward/src/allocation.rs

```diff
@@ -14,6 +23,34 @@ pub struct Connections {
     pub gravity_client: GravityQueryClient<Channel>,
 }
 
+pub fn format_eth_address(address: Address) -> String {
```

Comment:
Since the address was changed into a String, `format_eth_address` and `bytes_to_hex_str` aren't used any more, so let's delete them.